bolides/plotting.py

```diff
@@ -0,0 +1,180 @@
+import pandas as pd
+import numpy as np
+from .utils import reconcile_input
+from . import ROOT_PATH
+
+
+def plot_interactive(df, mode='earth', projection="eckert4",
```

Comment:
I tried running this and it errored due to plotly package not being available. Is this an unlisted dependency? I also presume this is experimental code because there is no documentation. If so, we should probably turn it off for the merge,

Reply:
Ah, I had to reinstall the package with `pip install -e .` to get `plotly` installed. OK, plot_interactive() "runs" however in my jupyter notebook it just displays a blank screen in the output cell. Should this do something?

Reply:
Checked on a fresh install in a new Conda environment and it seems to work. According to a comment [here](https://stackoverflow.com/questions/52771328/plotly-chart-not-showing-in-jupyter-notebook) maybe you need to completely restart the Jupyter server (not just the kernel) for plotly to work.

Reply:
Tried a fresh python environment install and `plot_interactive` now works.